src/shared/check_outdated_data.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int check_outdated_data(int maxTimeDiff, char *receivingData) {
+
+    time_t curTime; 
+
+    struct tm timeptr;
+
+    curTime = time(NULL);
```

Comment:
I would move this to line 7 so it reads
`time_t curTime = time(NULL);`

Reply:
yessir